src/components/SetImage.js

```diff
@@ -1,7 +1,7 @@
-export default function SetImage ({imgSrc, pt}){
+export default function SetImage ({imgSrc, pt, alt = ""}){
   return(
     <div className="set-image" style={{paddingTop: pt}}>
-      <img src={imgSrc} />
+      <img src={imgSrc} alt={alt} />
```

Comment:
I went ahead and added an `alt` tag to any image in the app. This is for accessibility reasons, it makes it so people with screen readers can view your app properly.